datafusion/physical-expr/src/aggregate/build_in.rs

```diff
@@ -232,54 +222,6 @@ mod tests {
         Ok(())
     }
 
-    #[test]
-    fn test_min_max_expr() -> Result<()> {
```

Comment:
We can move this test to slt

Reply:
should I move this test to SLT? I am not sure about the coercion part...

Reply:
I guess we can just remove it, since there is no special test here.